### PR TITLE
First day setup

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR I:

- installed react
- installed linters

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
```
Comment (HAMMAS-SALEEM) [inline src/index.css:18]: Capture

- [x] Kindly make your readme descriptive. It is the documentation of your project. So please take it seriously 😊 You can make your readme descriptive using this [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @MattGomb :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @MattGomb👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- App functions as it should.✔️

##

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes

- As stated by the previous reviewer, you need to have your work correctly documented. I can see you've made some changes but the README still has placeholder data from the given template.
Kindly replace all data with your personal/project data to make it look professional.

image

image


_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -68,3 +88,35 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
+- GitHub: [@MattGomb](https://github.com/MattGomb)
+- Twitter: [@MtysGombos1](https://twitter.com/MtysGombos1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gombos-m%C3%A1ty%C3%A1s-28139771/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: **[Optional]**

- [x] Kindly add the link of your issues page here to make your readme more descriptive 👍

`Feel free to check the [issues page](https://github.com/MattGomb/math-magicians/issues).`

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @MattGomb :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Gitflow
:heavy_check_mark: Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**